reports/evaluation/template-plot-forecasts.Rmd

```diff
@@ -0,0 +1,41 @@
+```{r prediction-plots, echo = FALSE, results='asis', fig.width = 6.5, fig.height=8}
+
+for (forecast_date in forecast_dates) {
+  cat(paste0("\n\n## ", forecast_date, "{.tabset .tabset-fade .tabset-dropdown} \n\n"))
+  
+  for (location in locations) {
+    cat("\n\n###", location, "{.tabset} \n\n")
+  
+    
+    for (target_variable in target_variables) {
+      cat("\n\n####", target_variable, "\n\n")
+      
+      filter_both <- list(paste0("target_variable %in% '", target_variable, "'"), 
+                          paste0("location_name %in% '", location, "'"))
+      filter_truth <- list(paste0("target_end_date > '", as.Date(forecast_date) - 7 * 10, "'"), 
+                           paste0("target_end_date <= '", as.Date(forecast_date) + 7 * 4, "'"))
+      filter_forecasts <- list(paste0("forecast_date == '", as.Date(forecast_date), "'"))
+      
+      
+      plot <- scoringutils::plot_predictions(data,
+                                             x = "target_end_date",
+                                             filter_both = filter_both,
+                                             filter_truth = filter_truth,
+                                             filter_forecasts = filter_forecasts,
+                                             facet_formula = ~ model,
+                                             # facet_formula = model ~ target_variable + location_name,
+                                             # facet_wrap_or_grid = "facet",
+                                             allow_truth_without_pred = FALSE,
+                                             scales = "free") + 
+        # ggplot2::ggtitle(paste0("Predictions for incident ", target_variable,  "s")) + 
+        ggplot2::theme(legend.position = "bottom", 
```

Comment:
can we have: 
- `expand_limits(y = 0)`
- `scale_y_continuous(lables = scales::comma)`
- `facet_wrap(ncol = 3)` (at the moment the plots look quite squashed when 4 columns are used)